src/utils/heatmapSpec.js

```diff
@@ -108,6 +225,7 @@ const spec = {
       encode: {
         enter: {
           cursor: { value: 'cell' },
+          tooltip: { signal: '{"Cell ID": datum.cellId, "Gene name": datum.gene, "Expression": format(datum.expression, ",.5f") }' },
```

Comment:
I think that rounding to `.5f` is too much here - `.3f` should be good, as it is for the embedding